src/PolykeyAgent.ts

```diff
@@ -68,6 +68,8 @@ class Polykey {
   // Proxies
   public readonly fwdProxy: ForwardProxy;
   public readonly revProxy: ReverseProxy;
+  public readonly ingressHost: string;
+  public readonly ingressPort: number;
```

Comment:
Do these need to be stored as internal state to `PolykeyAgent`? We introduce another copy of these here, when they can already be retrieved from the `ReverseProxy`. We should limit the number of state copies we make.

Reply:
I've just been replicating what was already done for the ports and hosts. But I can change them to getters that get it from the revProxy, etc if needed.

Reply:
I added getters that get the port and host from the revProxy clientServer etc but i'm keeping the above variables and making them private. we need them to know the desired ports when we start and stop networking.